docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 mysql:
-    image: autopilotpattern/mysql:latest
+    image: autopilotpattern/mysql
```

Comment:
Just eliminating the explicit `:latest` but at some point I need to figure out how to get the test rig to add the build identifier here so that we're testing the build we just created and not some arbitrary "latest"